src/js/common/components/AcceptTermsPrompt.jsx

```diff
@@ -17,12 +17,16 @@ class AcceptTermsPrompt extends React.Component {
     window.scrollTo(0, 0);
   }
 
-  onCancel(e) {
+  onCancel = (e) => {
     e.preventDefault();
     if (document.referrer !== '' && !this.props.isInModal) {
       browserHistory.goBack();
     } else {
-      browserHistory.push(this.props.cancelPath);
+      if (this.props.onCancel) {
+        this.props.onCancel();
+      } else {
+        browserHistory.push(this.props.cancelPath);
+      }
     }
   }
```

Comment:
I would confirm that this isn't in Jean Pants bc otherwise it may be moved to that repo

Reply:
Yes, this is in jeanpants; I'll make the changes there